backend/crypt/crypt.go

```diff
@@ -151,6 +151,14 @@ length and if it's case sensitive.`,
 				},
 			},
 			Advanced: true,
+		}, {
+			Name: "custom_suffix",
```

Comment:
I think we should probably just call this option `suffix`. Then the config will be

```
suffix = .bin
```

Which I think reads nicer than `custom_suffix` as all the config is custom isn't it?

Reply:
agreed, this has been updated.